draft-js-linkify-plugin/src/LinkAdd/index.js

```diff
@@ -0,0 +1,130 @@
+import React, { Component } from 'react';
+import ReactDOM from 'react-dom';
+import linkifyIt from 'linkify-it';
+import modifier from '../modifiers/addLink';
+import styles from './styles.css';
```

Comment:
this file should be renamed to `../linkAddStyles.css` in order to include it in `plugin.css` built with [concatCssFiles](https://github.com/draft-js-plugins/draft-js-plugins/blob/master/scripts/concatCssFiles.js)